lib/ansible.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from . import util, versions
+
+
+def install_deps():
+    """
+    Download and install any external dependencies required for Ansible to run.
+    """
+    # RHEL has rhc-worker-playbook
+    if versions.rhel.is_true_rhel():
+        # RHEL-7 seems to have it bundled in the base RPM
+        if versions.rhel <= 7:
+            return
+        pkg = 'rhc-worker-playbook'
+        ret = util.subprocess_run(['rpm', '--quiet', '-q', pkg])
+        if ret.returncode != 0:
+            util.subprocess_run(
+                ['dnf', '-y', '--nogpgcheck', 'install', pkg],
+                check=True)
+        # rhc-worker-playbook modules, exported per official instructions on
+        # https://access.redhat.com/articles/remediation
+        os.environ['ANSIBLE_COLLECTIONS_PATH'] = \
+            '/usr/share/rhc-worker-playbook/ansible/collections/ansible_collections/'
+
+    # CentOS and Fedora don't
+    else:
+        for collection in ['community.general', 'ansible.posix']:
+            util.subprocess_run(
+                ['ansible-galaxy', '-vvv', 'collection', 'install', collection],
+                check=True)
```

Comment:
CentOS7, same as RHEL7, `ansible` package has all available Ansible modules (including `community.general` and `ansible.posix` modules).

Reply:
Right, I'll make this applicable for 8+ then.